packages/twenty-server/src/core/workspace/workspace.entity.ts

```diff
@@ -54,6 +55,11 @@ export class Workspace {
   @OneToMany(() => User, (user) => user.defaultWorkspace)
   users: User[];
 
+  @OneToMany(() => UserWorkspace, (userWorkspace) => userWorkspace.workspace, {
+    onDelete: 'CASCADE',
+  })
```

Comment:
we don't want to delete workspace record when deleting related userWorkspaces record
```suggestion
  @OneToMany(() => UserWorkspace, (userWorkspace) => userWorkspace.workspace)
```